src/yang/ietf-babel.yang

```diff
@@ -596,32 +610,38 @@ module ietf-babel {
              logged.";
           reference
             "RFC ZZZZ: Babel Information Model, Section 3.3.";
-	}
+        }
 
         leaf packet-log {
-	  type inet:uri;
-	  config false;
+          type inet:uri;
+          config false;
           description
-	    "A reference or url link to a file that contains a
+            "A reference or url link to a file that contains a
              timestamped log of packets received and sent on
-             babel-udp-port on this interface. The [libpcap] file
+             udp port on this interface. The [libpcap] file
```

Comment:
It looks like the parameter name in this model is udp-port. I think the hyphen is needed.
s/udp port/udp-port/

Reply:
Ok